test/gtest/uct/test_md.cc

```diff
@@ -770,6 +770,17 @@ UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
                             UCT_MD_MEM_DEREG_FIELD_COMPLETION |
                             UCT_MD_MEM_DEREG_FIELD_FLAGS;
         status            = uct_md_mem_dereg_v2(md(), &params);
+        ASSERT_UCS_STATUS_EQ(UCS_ERR_INVALID_PARAM, status);
```

Comment:
why is it invalid param?
shouldn't it be invalid param in line 781?

Reply:
Because the rkey was not packed yet with `UCT_MD_MKEY_PACK_FLAG_INVALIDATE`, so nothing to invalidate.
Or I wrongly understood @shamisp 's comment?
what's wrong in line 781?

Reply:
ok